This file is part of mingw-cross-env.
See doc/index.html for further information.

Contains an ad hoc patch for cross building.

From 1ff5a73bee660f1d61f9cdd7c8d7aff29e959c4c Mon Sep 17 00:00:00 2001
From: mingw-cross-env
Date: Mon, 27 Sep 2010 12:00:52 +0200
Subject: [PATCH 1/2] fix .pc requirements


diff --git a/librsvg.pc.in b/librsvg.pc.in
index 372f199..b6ee7a9 100644
--- a/librsvg.pc.in
+++ b/librsvg.pc.in
@@ -9,7 +9,7 @@ css_supported=@CSS_SUPPORTED@
 Name: librsvg
 Description: library that renders svg files
 Version: @VERSION@
-Requires: glib-2.0 gio-2.0 gdk-pixbuf-2.0 cairo
+Requires: glib-2.0 gio-2.0 gdk-pixbuf-2.0 cairo libgsf-1 pangocairo libcroco-0.6
 Requires.private:
 Libs: -L${libdir} -lrsvg-@RSVG_API_MAJOR_VERSION@ -lm
 Cflags: -I${includedir}/librsvg-@RSVG_API_VERSION@
-- 
1.7.1
